src/makielayout/blocks.jl

```diff
@@ -67,7 +67,7 @@ macro Block(name::Symbol, body::Expr = Expr(:block))
 
         function Makie.default_attribute_values(::Type{$(name)}, scene::Union{Scene, Nothing})
             sceneattrs = scene === nothing ? Attributes() : theme(scene)
-            curdeftheme = deepcopy(CURRENT_DEFAULT_THEME)
+            curdeftheme = deepcopy(Makie.CURRENT_DEFAULT_THEME)
```

Comment:
```suggestion
            curdeftheme = deepcopy($(@__MODULE__).CURRENT_DEFAULT_THEME)
```
because it's not always guaranteed that the user will have loaded Makie directly

Reply:
Ah I guess you're right! I went with `$(Makie)` to make it clearer that it's not some other module that should get picked up....